features/financial_assistance/step_definitions/work_flow_steps.rb

```diff
@@ -36,14 +36,14 @@
   HbxProfile.any_instance.stub(:under_open_enrollment?).and_return(true)
 end
 
-And(/^it is before open enrollment$/) do
-  oe_start_on = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.detect {|bcp| bcp.start_on.year == Date.today.year}.open_enrollment_start_on
-  allow(TimeKeeper).to receive(:date_of_record).and_return Date.new(oe_start_on.year,10,27)
+And(/^current date is between renewals generation date and open enrollment start on$/) do
+  oe_start_on = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.detect { |bcp| bcp.start_on.year == Date.today.year }.open_enrollment_start_on
+  allow(TimeKeeper).to receive(:date_of_record).and_return Date.new(oe_start_on.year, 10, 27)
 end
 
-And(/^it is after open enrollment$/) do
+And(/^current date is outside renewals generation date and open enrollment start on$/) do
   oe_end_on = HbxProfile.current_hbx.benefit_sponsorship.benefit_coverage_periods.last.open_enrollment_end_on
-  allow(TimeKeeper).to receive(:date_of_record).and_return Date.new(oe_end_on.year,2,15)
+  allow(TimeKeeper).to receive(:date_of_record).and_return Date.new(oe_end_on.year, 2, 15)
```

Comment:
Similar note as above-- there's probably no chance that the OE end date would be after 2/15, but that date is driven by a setting and this one is hardcoded.